ledger/src/shred.rs

```diff
@@ -786,6 +786,22 @@ pub mod layout {
             .ok_or(Error::InvalidPayloadSize(shred.len()))
     }
 
+    pub fn is_retransmitter_signed(shred: &[u8]) -> Result<bool, Error> {
```

Comment:
This name is a bit misleading because the function only checks if the shred is of resigned variant.
But that does not guarantee that the retransmitter has actually attached a valid signature here.